modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -1,40 +1,50 @@
+While it is a bit more secure than a central encryption approach, User key-based encryption has some disadvantages.
+It blocks some additional functions such as the integration of an online editor like LibreOffice or OnlyOffice into ownCloud and can cause problems when sharing files with groups.
+Therefore Master-key-based encryption is now the recommended setup for all new installations.
 
-If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition.
-These are:
+The decryption workflow described here works for both MySQL and SQLite based databases. However, it will only work with the following pre-conditions:
+
+ - the admin recovery password needs to be activated during creation of new accounts for all users (settings, personal, encryption).
+  - the admin recovery password must be  available for the ownCloud administrator
+
+It is possible that some or all users see a re-synchronisation of their data from the server to the desktop client - especially for shared folders.
+A notification to the users (e.g. through the https://marketplace.owncloud.com/apps/announcementcenter[announcement app]) prior to the migration process is recommended.
+
+If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition:
 
 . xref:disable-user-key-based-encryption[Disable User Key-based encryption]
 . xref:remove-the-encryption-records-from-the-owncloud-database[Remove the encryption records from the ownCloud database]
 . xref:remove-the-files_encryption-directory[Remove the `files_encryption` directory]
 . xref:encrypt-the-filesystem-using-master-key-based-encryption[Encrypt the filesystem using Master Key-based encryption]
 
-include::{partialsdir}/configuration/server/enable-single-user-mode.adoc[leveloffset=+1]
 
 [[disable-user-key-based-encryption]]
 == Disable User Key-based Encryption
 
-The first part of the migration process is to take is to decrypt all files and to disable encryption in ownCloud, which requires three commands to be executed.
+The first part of the migration process is to decrypt all files and to disable encryption in ownCloud, which requires three commands to be executed.
 These commands are: xref:configuration/server/occ_command.adoc#encryption[`occ encryption:decrypt-all` and `occ encryption:disable`], and xref::configuration/server/occ_command.adoc#apps-commands[`occ app:disable`].
 
 You can see an example of calling the commands listed below, configured to require no user interaction.
 
 [source,console]
 ----
-sudo -u www-data occ encryption:decrypt-all --continue=yes && \
-  sudo -u www-data occ encryption:disable --no-interaction && \
-  sudo -u www-data occ app:disable --no-interaction encryption
+sudo -u www-data php occ encryption:decrypt-all --continue=yes && \
```

Comment:
{occ-command-example-prefix}
https://github.com/owncloud/docs/blob/master/docs/best-practices.md#occ-examples

Reply:
Thanks. I forgot to think about this.